### ISSUE TITLE
[REQUEST] Yandex AppImage

### ISSUE THREAD

**Issue by in4matix** [ivan-hc/AM#760]
Hi Ivan,

You ever try to make Yandex web browser AppImage from official DEB ?

**Comment by ivan-hc** [comment]
It's better if I move this issue here, maybe you or anyone else will want to try creating and maintaining Yandex browser AppImages and then adding them to the "AM" package manager database.

**Comment by ivan-hc** [comment]
If anyone asks why I don't want to maintain this, is because:
1. I've already 70 AppImages to maintain, this package manager and my AppImage helpers;
2. Also if my name is "Ivan", I'm italian, and I don't talk nor understand Russian, so as to avoid people asking me questions in Russian. Better not to fuel this misunderstanding.

**Comment by ivan-hc** [comment]
@in4matix I finally did it https://github.com/ivan-hc/Yandex-Browser-appimage :face_with_head_bandage: 

It's just because I don't want to waste all the time and work I put into it.

**Comment by ivan-hc** [comment]
All three versions are available in the database of "AM", the stable...
```
am -i yandex-browser
```
...beta...
```
am -i yandex-browser
```
...and corporate.
```
am -i yandex-browser
```
I don't know what is the difference, use the one you prefer.

PS: also if I create and distribute unofficial AppImages of [Google Chrome](https://github.com/ivan-hc/Chrome-appimage), [Opera](https://github.com/ivan-hc/Opera-appimage), [Vivaldi](https://github.com/ivan-hc/Vivaldi-appimage), [MS Edge](https://github.com/ivan-hc/MS-Edge-appimage) and [Chromium](https://github.com/ivan-hc/Chromium-Web-Browser-appimage), I am a Firefox user and a strong supporter of  Mozilla Firefox.

If it were up to me I would abolish all browsers based on Chromium. But I enjoy creating AppImage packages, to be honest, and Electron-based ones are the easiest to create, by far. It's maintaining them that's problematic, and as I said above, I have too many of them.

My luck is that Edge, Chrome, Opera, Vivaldi, Yandex... but also Skype... are all built in the exact same way: just move the entire contents of the app directory to the root of the AppImage, add an AppRun, an icon, the .desktop file... and the AppImage is ready to use.

In short, creating Chromium/Electron-based AppImages is jocke. Anyone can do it.

In fact, I'm still waiting for [Bottles to be created in Electron](https://usebottles.com/blog/bottles-next-a-new-chapter/), I could easily build even one of the most popular Flathub Flatpaks. [My current attempt](https://github.com/ivan-hc/Bottles-appimage) is a mess to maintain.

**Comment by in4matix** [comment]
I also prefer Firefox.
